parent_models_to_solr_field.xslt

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:string="xalan://java.lang.String"
+  xmlns:encoder="xalan://java.net.URLEncoder">
+
+  <xsl:template name="perform_ri_query">
```

Comment:
Kind... of thinking we could just use the [`perform_traversal_query` template](https://github.com/discoverygarden/islandora_transforms/blob/8e9bbde3d05ee3a1056c704b963162e95137d0da/library/traverse-graph.xslt#L93-L103)... only thing it doesn't handle that this does is the PID replacement; however... may be a moot point, if we do move to a full traversal...